google/cloud/bigtable/examples/README.md

```diff
@@ -3,25 +3,36 @@
 This directory contains minimal examples to get started quickly, and are
 referenced from the Doxygen landing page as "Hello World".
 
-## Compiling the Examples
+## Table of Contents
+
+- [Before you begin](#before-you-begin)
+  - [Compile the Examples](#compile-the-examples)
+  - [Run the examples against the Cloud Bigtable Emulator](#run-the-examples-against-the-cloud-bigtable-emulator)
```

Comment:
I think we should remove the code that suggests using the emulator.  It is good for development, but a bad example to give our users.